Tasks/Maven/maventask.ts

```diff
@@ -318,7 +320,7 @@ function publishCodeCoverage(isCodeCoverageOpted: boolean) {
 }
 
 function publishCCToTfs() {
-    if (pathExistsAsFile(summaryFile)) {
+    if (tl.exist(summaryFile)) {
```

Comment:
tl.exist(summaryfile) && tl.stats(summaryFile).isFile()

Do this every where where you replaced